kornia/geometry/ransac.py

```diff
@@ -0,0 +1,224 @@
+"""Module containing RANSAC modules."""
+import math
+from typing import Optional, Tuple, Union
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from kornia.geometry import (
+    find_fundamental,
+    find_homography_dlt,
+    find_homography_dlt_iterated,
+    symmetrical_epipolar_distance,
+)
+from kornia.geometry.homography import symmetric_transfer_error
+
+__all__ = ["RANSAC"]
+
+
+class RANSAC(nn.Module):
+    """Module for robust geometry estimation with RANSAC. https://en.wikipedia.org/wiki/Random_sample_consensus.
+
+    Args:
+        model_type (str): type of model to estimate, e.g. "homography".
+        batch_size (int): number of generated samples at once .
+        max_iter (int): maximum batches to generate. Actual number of models to try is batch_size * max_iter.
+        inl_th (float): threshold for the correspondence to be an inlier
+        confidence (float): desired confidence of the result, used for the early stopping.
+        max_lo_iters (int): number of local optimization (polishing) iterations
+
+    Returns:
+        - estimated model, shape of :math:`(1, 3, 3)`.
```

Comment:
```suggestion
        - estimated model, shape of :math:`(3, 3)`.
```
in the tests you don't return batch. I guess doesn't make sense here batch